NetcoinLib/Services/CustomerService.cs

```diff
@@ -0,0 +1,51 @@
+﻿using NetcoinLib.Models;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace NetcoinLib.Services
+{
+    public class CustomerService
+    {
+        private INetcoinRepository repository;
+        public CustomerService(INetcoinRepository repository)
+        {
+            this.repository = repository;
+        }
+        public bool CreateCustomer(string Name, string LegalId, string Area = null, string Address = null, string PostalCode = null)
```

Comment:
It's worth considering that if Area, Address and PostalCode is null, will it properly serialise to the text files? In addition, there are more properties in Customer (country, city, phonenumber)